app/flatpak-builtins-utils.c

```diff
@@ -357,8 +360,19 @@ flatpak_resolve_duplicate_remotes (GPtrArray    *dirs,
   if (out_dir)
     {
       if (dirs_with_remote->len == 0)
-        return flatpak_fail_error (error, FLATPAK_ERROR_REMOTE_NOT_FOUND,
-                                   "Remote \"%s\" not found", remote_name);
+        {
+          if (!opt_user && !opt_system && !opt_installations)
+              return flatpak_fail_error (error, FLATPAK_ERROR_REMOTE_NOT_FOUND,
+                                         _("Remote \"%s\" not found. Use flatpak remote-add to add a remote"),
+                                         remote_name);
+          else
+            {
+              const char *inst_option = opt_user ? "--user" : (opt_system ? "--system" : "--installation");
+              return flatpak_fail_error (error, FLATPAK_ERROR_REMOTE_NOT_FOUND,
+                                         _("Remote \"%s\" not found. Perhaps the wrong installation was specified with %s?"),
```

Comment:
This message seems a bit forced. Maybe something like:

Remote "flathub" not found in the [user|system|'foo'] installation. 

Also, If we wanna be *real* AI we could look for matching remotes in other remotes and mention those in the error.

Reply:
Updated the branch accordingly. I didn't bother to check other installations automatically.